react/react_and_the_backend/project_messaging_app.md

```diff
@@ -7,17 +7,18 @@ Think of your favorite messaging app - Discord, Telegram, WhatsApp, Facebook Mes
 <div class="lesson-content__panel" markdown="1">
 
 1. Keep in mind that the apps we mentioned earlier have entire teams of developers working on them, so you shouldn't feel the need to go overboard with features in your app. At a minimum, though, your app should include the following core functionality:
-    1. Authorization
-    1. Sending messages to another user
-    1. Customizing a user profile
+   1. Authorization
+   1. Sending messages to another user
+   1. Customizing a user profile
 1. Spend some time planning out your app. What will the user interface look like? What will the data model look like? What libraries will you need to use?
 1. Start building your app by implementing the core functionality in your backend and frontend.
+   1. You may have realised that a REST API backend cannot handle real time updates. It is "request-response", so the server can only respond to a request. If a user sends a message to another user, a REST API cannot automatically notify the recipient as the recipient will not have requested that data. Whilst there are ways to implement real time functionality (e.g. polling, server-sent events, websockets etc.), these methods have not been taught and so you are not expected to implement any in this app.
```

Comment:
```suggestion
   1. You may have realized that a REST API backend cannot handle real time updates. It is "request-response", so the server can only respond to a request. If a user sends a message to another user, a REST API cannot automatically notify the recipient as the recipient will not have requested that data. Whilst there are ways to implement real time functionality (e.g. polling, server-sent events, websockets etc.), these methods have not been taught, so you are not expected to implement any in this app.
```
"realize" for "realise" is an American English vs UK English thing. The style guide indicates a preference for American English.